src/openrct2/park/ParkFile.cpp

```diff
@@ -417,32 +417,32 @@ namespace OpenRCT2
 
         void ReadWriteScenarioChunk(GameState_t& gameState, OrcaStream& os)
         {
-            os.ReadWriteChunk(ParkFileChunkType::SCENARIO, [&os](OrcaStream::ChunkStream& cs) {
-                cs.ReadWrite(gScenarioCategory);
-                ReadWriteStringTable(cs, gScenarioName, "en-GB");
+            os.ReadWriteChunk(ParkFileChunkType::SCENARIO, [&os, &gameState](OrcaStream::ChunkStream& cs) {
```

Comment:
nitpicky, but I'd go with the same order as the aguments of this function :P

```suggestion
            os.ReadWriteChunk(ParkFileChunkType::SCENARIO, [&gameState, &os](OrcaStream::ChunkStream& cs) {
```